.github/ISSUE_TEMPLATE/user-story-issue.md

```diff
@@ -0,0 +1,23 @@
+---
+name: User Story
+about: This template provides a basic structure for user story issues.
+labels: Needs Triage
+---
+
+# Description
+As a ..., I want to ..., so I can ...
+
+<Link to Design Document or User Story>
```

Comment:
Is this supposed to be `User Story` or something else (like `Epic`)?

If it is supposed to be `User Story` can we be more specific as to what type of `User Stories` we expect to link to other `User Stories`?  Like `GreenSoftwareFoundation repo User Story`